<?xml version="1.0" encoding="UTF-8"?>

<launch>
     <node pkg="image_view" type="image_view" name="image_processed">
    	  <remap from="image" to="camera_hsv"/>
    </node>
</launch>
